app/src/main/java/com/arygm/quickfix/ui/search/ProfileResults.kt

```diff
@@ -0,0 +1,108 @@
+package com.arygm.quickfix.ui.search
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyListState
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.arygm.quickfix.MainActivity
+import com.arygm.quickfix.R
+import com.arygm.quickfix.model.account.Account
+import com.arygm.quickfix.model.account.AccountViewModel
+import com.arygm.quickfix.model.profile.WorkerProfile
+import com.arygm.quickfix.model.search.SearchViewModel
+import com.arygm.quickfix.ui.theme.poppinsTypography
+import com.arygm.quickfix.utils.LocationHelper
+
+@Composable
+fun ProfileResults(
+    modifier: Modifier = Modifier,
+    profiles: List<WorkerProfile>,
+    listState: LazyListState,
+    searchViewModel: SearchViewModel,
+    accountViewModel: AccountViewModel,
+    widthRatio: Float,
+    heightRatio: Float,
+    onBookClick: (WorkerProfile) -> Unit
+) {
+  // Column for wrapping the list
+  Column(modifier = Modifier.fillMaxWidth(), horizontalAlignment = Alignment.Start) {
+    Text(
```

Comment:
I don't think this text is even necessary. What we could do instead is that when there's a search query, the topAppBar (with the two buttons Seach and Announce) disappears to leave only the search bar and the results, I think that would be cleaner 😄.

Reply:
Indeed I have removed the Text, However I believe it makes sense to keep the **search** and **announce** components visible even when actively searching, for the following reasons:

1. **UI Consistency**:  
   Removing the component would either push the search bar to the top of the screen or leave an awkward gap, which I find visually unappealing and disruptive for the user experience.  

2. **Logical Placement**:  
   The search bar directly relates to searching, so it makes sense to keep the top button visible to indicate an active search context.

3. **Usability**:  
   Users might still want to click the **announce** button while searching. Hiding it forces them to clear their search first, which is unintuitive.

4. **Technical Consideration**:  
   The current file structure (`SearchOnboarding.kt` within `QuickFixFinder.kt`) is not optimized for easily changing this behavior. It would require additional refactoring.

Let me know if you disagree or have other suggestions—I'm happy to make changes! 😊